System.IO.Abstractions.TestingHelpers.Tests/WindowsSpecifics.cs

```diff
@@ -9,5 +9,9 @@ internal static class WindowsSpecifics
         public const string UNCPaths = "UNC paths are a Windows-only concept";
         
         public const string StrictPathRules = "Windows has stricter path rules than other platforms";
+
+        public const string EmptyInvalidPathChars = "Path.GetInvalidPathChars() doesn't return anything on Mono";
```

Comment:
I guess it's not actually Windows-specific, it's Mono-exclusive. Add an `IngoreMono` attribute that ignores test when running on Mono?

Reply:
Hm, unsure - we don't actually test on Mono - does the test that uses this also fail on .NET Core on Linux?

Reply:
Let me give it a try...

Reply:
OK, so the test in question fails on Linux in .NET Core, because Linux paths can have tab chars in the name. I'm going to label that test `[WindowsOnly(WindowsSpecifics.StrictPathRules)]`. `GetInvalidPathChars` just returns `\0` on Linux, so we'll say this is a windows-only test.